src/pipecat/transports/services/daily.py

```diff
@@ -62,6 +62,9 @@
         VirtualCameraDevice,
         VirtualSpeakerDevice,
     )
+    from daily import LogLevel as DailyLogLevel
+
+    LogLevel = DailyLogLevel
```

Comment:
Indentation?

Also...seems like almost everything else exported from this file is prefixed with `Daily`, so why bother mapping it to `LogLevel`?

Reply:
I am leaving it as DailyLogLevel as you suggested. Makes sense. 👍